### PR TITLE
tweak skateboard crashes, gives brain damage

### PR DESCRIPTION
**Pull request by checkraisefold**
:cl: zennerx
tweak: Skateboard crashes now give slight brain damage!
rscadd: Using a helmet prevents brain damage from the skateboard!
/:cl:

this balances skateboard crashes and does what they do in real life; give brain damage. but on a game it's much less worse.

### CODE REVIEW

**Comment by Anonmare** [comment]
Tubular!

**Comment by CosmicScientist** [comment]
Radicool!

**Comment by TribeOfBeavers** [comment]
Could you have it so wearing a helmet or hardhat prevents it?

**Comment by praisenarsie** [comment]
^^^^^^^^^^

**Comment by PKPenguin321** [comment]
Pretty funny

**Comment by checkraisefold** [comment]
done @TribeOfBeavers @praisenarsie

File: code/modules/vehicles/scooter.dm [hunk 1]
```diff
@@ -57,6 +57,9 @@
 		H.throw_at(throw_target, 4, 3)
 		H.Knockdown(100)
 		H.adjustStaminaLoss(40)
+		if(!istype(H.head,/obj/item/clothing/head/helmet/) && !istype(H.head,/obj/item/clothing/head/hardhat))
```
Comment (Jordie0608) [inline code/modules/vehicles/scooter.dm]: Instead of checking for these types maybe check for melee armor on any helmet type?

Comment (checkraisefold) [reply code/modules/vehicles/scooter.dm]: we really just want -normal- helmets and hardhats, kind of like a bike helmet. if I did that, literally would have people wearing cake hats to stop brain damage even though they're not strong enough

Comment (Jordie0608) [reply code/modules/vehicles/scooter.dm]: Well the cake hat in specific has no armor so it wouldn't but I know what you mean.

I made myself a list of all the hats and their armor; turns out there's a few weird cases whichever sort of check you go with so just roll with this it's close enough to fine.

**Pull request merged**